packages/rainbowkit/src/wallets/walletConnectors/walletConnectWallet/walletConnectWallet.ts

```diff
@@ -1,47 +1,32 @@
-import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
-import { getWalletConnectUri } from '../../../utils/getWalletConnectUri';
-import { isIOS } from '../../../utils/isMobile';
+import { WalletConnectParameters } from 'wagmi/connectors';
 import { Wallet } from '../../Wallet';
 import { getWalletConnectConnector } from '../../getWalletConnectConnector';
-import type { WalletConnectConnectorOptions } from '../../getWalletConnectConnector';
 
 export interface WalletConnectWalletOptions {
   projectId: string;
-  chains: Chain[];
-  options?: WalletConnectConnectorOptions;
+  walletConnectParameters?: WalletConnectParameters;
 }
 
+// Used for maintance purposes for `connectorsForWallets` logic
+// wcId = "WalletConnect id"
+export const wcId = 'walletConnect';
+
```

Comment:
Sometimes when we use the same variable for comparing and updating stuff like [here](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/components/ConnectOptions/MobileOptions.tsx#L87) and [here](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/wallets/connectorsForWallets.ts#L134). Just wanna make sure it's maintainable.

Otherwise if we're going to do more features with `walletConnect` we might introduce some bugs by accident so want to make sure that `id` is being shared across all over the app.